website/static/js/pages/register-page.js

```diff
@@ -17,35 +18,32 @@ $(function() {
     // opt into tooltip
     $('[data-toggle="tooltip"]').tooltip();
 
-    var viewModel;
-    var selector;
-    var editor = new registrationUtils.RegistrationEditor({
-        schemas: '/api/v1/project/schemas/',
-        create: node.urls.api + 'drafts/',
-        submit: node.urls.api + 'drafts/{draft_pk}/submit/',
-        update: node.urls.api + 'drafts/{draft_pk}/',
-        get: node.urls.api + 'drafts/{draft_pk}/',
-        draftRegistrations: node.urls.web + 'registrations/'
-    }, null, true);
-
-    if (ctx.draft) { // if registering draft
+    // if registering draft
```

Comment:
I refactored this block some and it looks like it's been rolled back. Any reason for this?